### DISCUSSION TITLE
Disable colors in log output

### DISCUSSION THREAD

**Discussion by ghost** [postalserver/postal#2905] [Feature suggestions]
How do I prevent color codes from being used in logging? 

Up until now, I used "docker logs postal-web-1" for example to view logs, so it didn't really bug me. However, I wanted to be able to use the logs outside of the container - I chose to implement the syslog driver in docker, and output to /var/log/postal-web.log, (also through rsyslog.d)  but the color codes are coming through. Even if I use something like lnav to view it, I still see the color codes, so they are being partially interpretted. Is there some way to have it log without the color codes?

**Comment by willpower232** [comment]
I don't really know what I'm talking about but does this help? https://stackoverflow.com/a/52862428

**Reply by ghost** [reply]
that worked!! I used individual conf files in /etc/rsyslogd so it only affected postal, I can now see the colors in the main log, which is still better than partial colors.

**Comment by adamcooke** [comment]
It is not possible to disable these entirely right now. It would be nice though. Moved to Feature Suggestions.

**Reply by ghost** [reply]
Thanks!
